src/popup.html

```diff
@@ -1,170 +1,304 @@
 <!DOCTYPE html>
```

Comment:
I did not actually review this file since the formatting changes make it hard to see what's actually new. If you could point out which parts you changed, I'd be happy to give it another look :)

Reply:
@Vinnl This comment was what made me a closer look on my review. Thanks for making this note!